deno/send_message/src/mod.ts

```diff
@@ -0,0 +1,49 @@
+import send_email_mailgun from "./functions/send_email_mailgun.ts";
+import send_sms_twilio from "./functions/send_sms_twilio.ts";
+import send_message_discord_webhook from "./functions/send_message_discord_webhook.ts";
+import send_tweet from "./functions/send_tweet.ts";
+
+export default async function (req: any, res: any) {
+  console.log(req);
+  console.log(res);
+  const type: string = req.payload["type"];
```

Comment:
Ideally, `req.payload` should be a JSON string and the function should use `JSON.parse()` to parse the string so that this function can be compatible with Appwrite.

Reply:
ok

Reply:
When I use JSON.parse() It returns an error 

Without JSON.parse it is working


![image](https://user-images.githubusercontent.com/88965873/201272586-f354a4b3-4932-4b4d-86be-abddb9974ef7.png)
I think openruntimes handles it